src/lib/gamepad.ts

```diff
@@ -1,8 +1,7 @@
 import { Button } from 'react-gamepad'
 import mod from '../utils/mod'
 
-export const getActiveElement = () =>
-  document.activeElement! as HTMLInputElement
```

Comment:
Nothing was relying on this being an `HTMLInputElement`, and it typically won't be, so I just made it an `HTMLElement`.

Reply:
Yeah, this was related to legacy usage.